cmd2/transcript.py

```diff
@@ -0,0 +1,213 @@
+#
```

Comment:
Maybe add a brief comment regarding the overall purpose of the code in this file?